### ISSUE TITLE
AM-softmax implement details

### ISSUE THREAD

**Issue by sweetTT** [CoinCheung/pytorch-loss#5]
when use am-softmax loss, do we need to add another fully connect layer before using AM-softmax loss?

such as Avg_pooling-->fc-->amsoftmax?

**Comment by CoinCheung** [comment]
That would depend on the dimension of you features. Amsoftmax loss is used to train embedding networks, if your feature embeddings have dimension different from the avg pooling, you should add another fc layer to adjust the embedding dimension.

**Comment by CoinCheung** [comment]
I am closing this. You can still leave a message if you have more to discuss.
